src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -52,11 +66,13 @@ public KafkaMonitor(Map<String, Map> testProps) throws Exception {
         _services.put(name, service);
       }
     }
-
-    _executor = Executors.newSingleThreadScheduledExecutor();
   }
 
-  public void start() {
+  public synchronized void start() {
+    if (_isShutdown) {
```

Comment:
If we assume that `start()` can be called only once, and we will only call `stop()` or `awaitShutdown()` after we have called `start()`, then it seems that `_isShutdown` should always be false when it is checked in `start()`, right? In other words we don't need this check.